examples/kubernetes/1.10/cilium-ds.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     spec:
       serviceAccountName: cilium
       containers:
-      - image: cilium/cilium:v1.0
+      - image: cilium/cilium:v1.0.0-rc13
```

Comment:
Why do we need to update this tag? Will we have to update this with every subsequent RC?

Reply:
@ianvernon Yes, because the image pull policy is defined as `IfNotPresent`. If it was kept to v1.0, which changes every time, as the v1.0 branch is updated, users would never got the most recent image

Reply:
Ah, makes sense. Thanks!